content/en/news/2019-03-25-server-outage-aftermath.md

```diff
@@ -0,0 +1,139 @@
+important thing first: the server is back online, and we have brought
+back online all services already. Users of the forum interface on
+lists.secretchronicles.org or the tsc-devel mailing list should pay
+close attention to the section [Current State](#current-state) below
+as action is required.
+
+## What happened?
+
+On March 12<sup>th</sup>, 2019, our server
+`alexandria.secretchronicles.org` became completely unresponsive. It
+neither served web pages, nor could we log in via SSH as the TCP
+connections just timed out. alexandria was a VPS server running on an
+OpenVZ host and was sponsored by [First-Root UG][2], Germany. quintus
+quickly get into contact with them by raising a support ticket on that
+day and they diagnosed that the server had “hung up” and could not be
+rescued. As an alternative, First-Root offered to provide us with a
+KVM server on March 13<sup>th</sup>, 2019. We considered all data on
+the server as lost and checked our backups.
+
+As it turns out, we had just a few months before upgraded the
+operating system from Debian 8 to Debian 9. Before that upgrade, we
+had automatic backups in use, but due to time constraints we didn't
+set them up yet on Debian 9, although we intended to do so. Murphy's
+law has no mercy, so the last backup we had available was from October
+2018. Given the low activity of the project in the time since October
+2018, this isn't too much of a loss, though. The source code of TSC
+was entirely unaffected anyway, because it is stored in Git, every
+developer had a recent copy, and it is available completely [via
+GitHub][9] anyway. **We have not lost any source code**.
+
+It was decided that the old VPS would not be set up again, because
+that would duplicate work for the TSC team, which is running low on
+free time anyway. Instead, we wanted to wait for First-Root to provide
+the KVM machine. During this time, which we expected to be short, all
+services hosted on the secretchronicles.org domain would be
+unavailable. On March 13<sup>th</sup>, 2019, quintus [opened a ticket][3] on the
+bugtracker on GitHub to announce the problem publically as the
+tsc-devel mailing list was unavailable due to the outage.
+
+At this point, all communication between the TSC team members
+basically happened in our IRC channel `#secretchronicles` on
+freenode. Our IRC bot, furbot, was also hosted on that server, which
+means that for this time no public chat logs are available on
+[chatlogs.secretchronicles.org][4].
+
+Despite some status queries, First-Root did not reply to the support
+ticket raised with them after March 13<sup>th</sup> anymore, nor did
+they provide the promised KVM server. By March 24<sup>th</sup>, 2019,
+we had been nearly two weeks offline and we decided that we need to
+venture for an alternative. TSC team member xet7 is involved in the
+[Wekan project][5], an open-source kanban board implementation. The
+Wekan project had a spare server they currently have not in use and
```

Comment:
> Wekan project had a spare server they currently have not in use and

change to:
Wekan project has a spare server that is used to [port Wekan to arm64](https://blog.wekan.team/2018/01/wekan-progress-on-x64-and-arm/index.html)